test/basic/checking-fail-bpl.c

```diff
@@ -0,0 +1,17 @@
+#include <stdio.h>
```

Comment:
The convention I have been using is that failing regressions should have file names ending in `_fail`. It would be good to stick with that.